base/regex.jl

```diff
@@ -396,9 +396,30 @@ function findall(t::Union{AbstractString,AbstractPattern}, s::AbstractString; ov
     return found
 end
 
+"""
+    findall(c :: AbstractChar, s :: AbstractString)
```

Comment:
Don't put spaces around `::`.

(In general, mimic the style of the surrounding code.)

Reply:
Done (I tried to be obsessive about that, but that one skipped).